### ISSUE TITLE
Comments pushed from VAR does not have indent

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#1480]
Something similar was reported by some of the users - I have noticed that if we're formatting VAR with SplitTooLongLine formatter, which causes comments to be 'pushed', indent is lost:

```
    VAR    ${x}  # robocop: off=some-rule
    ...    verylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylongline  # some comment
```

is formatted to

```
# robocop: off=some-rule
    VAR    ${x}
    ...    verylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylonglineverylongline
```
